recipes/libm2k/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "libm2k" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/analogdevicesinc/{{ name }}/archive/v{{ version }}.tar.gz
+    sha256: 64f62cf9a72d625c4a4344a1984a2f8c5d4a95aeaed0026a0cba7ac0c0ca7390
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('libm2k', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - {{ cdt('libudev-devel') }}  # [linux]
+    - make  # [not win]
+    - ninja  # [win]
+  host:
+    - libiio
+    - libusb >=1
+    - python
+    - swig
+    - setuptools
+    - icu  # [osx]
+    - libiconv  # [osx]
+    - xz  # [osx]
+    - zlib  # [osx]
+  run:
+    - libiio
+    - python
+  run_constrained:
+    # require the MACOSX_DEPLOYMENT_TARGET set in conda_build_config.yaml
+    # (10.11 to be able to compile bonjour support)
+    - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}  # [osx]
```

Comment:
Probably not necessary since the build looks like it is working fine with the default 10.9 SDK anyway.